LPD-50406 [FE] Add FDS Content Renderers

oliv-yu opened the pull request:
# What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-50406 - [FE] Add FDS Content Renderers

This applies the content renderers for Name, Author, Type, and Space with the extra styling:
- Name with a lock icon (lock appears if System property is true)
- Type with the name translated
- Author with the user icon
- Status shown as colored tags
- Space currently has the placeholder of Scope and should be updated after the Space API is added. It needs to know whether it is a Structure or Content, and it will need a way to color-coordinate the different spaces. Right now it's just adding styling for the two options, but let me know if I should make any changes for it!

<img width="1579" alt="image" src="https://github.com/user-attachments/assets/df97746d-3f8e-4c86-8b4c-b39368b821f9" />

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-portal-source-format/2794/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/2794/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/877521a4d8ffc19a78a68b5ecfc3289ae395b984">877521a4d8ffc19a78a68b5ecfc3289ae395b984</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-50406-cms-fds-renderers">LPD-50406-cms-fds-renderers</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/aeb8ffef21d2e5145e51113990ae283048e89f62">aeb8ffef21d2e5145e51113990ae283048e89f62</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-source-format/2794/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/2794/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-2.liferay.com/job/test-portal-source-format/2794/">test-portal-source-format#2794</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/2794/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6444">liferay-content-management#6444</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/158599098">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6444 - 2025-03-06[17:32:11]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>158599098</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-2.liferay.com/job/publish-testray-report/5323/">publish-testray-report#5323</a><br/></div></div>

ambrinchaudhary commented on modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/FDSPropsTransformer/cell_renderers/SpaceRenderer.tsx:
```diff
@@ -0,0 +1,33 @@
+		</span>
+	) : (
+		<span className="align-items-center d-flex">
+			<Sticker
```
Here I would suggest to use the new SpaceSticker (it was merged [here](https://github.com/brianchandotcom/liferay-portal/pull/159652)). Each space would have its own color in the future, so that component will handle the stickers. 
The only change needed would be to add a size property to that class, I was not aware of that need.

oliv-yu replied:
Thanks for making that component! Will apply it for the cell renderer then

ambrinchaudhary commented on modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/FDSPropsTransformer/cell_renderers/NameRenderer.tsx:
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const NameRenderer = ({
+	itemData,
```
Here I'm just curious, where this `itemData` comes from?

marcogalluzzi replied:
About this, is it possible to remove the System column in the table? I suppose is needed in `itemData`, so is there a way to hide it?

oliv-yu replied:
Sounds good! I'll go ahead and remove the System column in the table from StructuresSectionTableFDSView. 👍 

I think `itemData` is one of the props passed into table cell renderers where it contains all the information of that particular line item. It was being used in several examples for frontend-data-set (was looking around [here](https://github.com/liferay/liferay-portal/blob/540d6a1d49322248ffb2591a5ca77e02f8f40103/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/CustomAuthorTableCell.tsx#L23) which really helped in knowing how to apply it!).

ambrinchaudhary reviewed (changes requested):
Great work here @oliv-yu! I just have one suggestion, which is using the SpaceSticker component if you think is possible. 
The rest looks perfect ❤️

marcogalluzzi reviewed (approved):
Nice work. I've added more records to the table to test it.

![image](https://github.com/user-attachments/assets/4f763a62-c8b3-4c21-913e-72b5cd968728)

oliv-yu commented:
Thanks for all the feedback @ambrinchaudhary @marcogalluzzi ! I added the changes and rebased on master, also cherry-picked the SpaceSticker updates from https://github.com/liferay-content-management/liferay-portal/pull/6448 so that it's importing the most recent SpaceSticker.

Before reindexing, it looked like the default empty state had no margin, ended up adding an extra commit to update 😅 
<img width="1789" alt="Screenshot 2025-03-07 at 12 44 19 PM" src="https://github.com/user-attachments/assets/9ec8e4e6-4697-4f34-abbf-34319044d615" />

Let me know if it looks alright, thanks again!

ambrinchaudhary commented:
ci:test:sf

ambrinchaudhary commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12823/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12823/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-39.liferay.com/job/test-portal-source-format/7646/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-source-format/builds/7646/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-source-format/7634/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/7634/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7747cb14a86dba4def878d1ef176ff42f8f274d">f7747cb14a86dba4def878d1ef176ff42f8f274d</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-50406-cms-fds-renderers">LPD-50406-cms-fds-renderers</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/3a21d44e64e758b7707d3a237d0c7adac67b3e00">3a21d44e64e758b7707d3a237d0c7adac67b3e00</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-39.liferay.com/job/test-portal-source-format/7646/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-source-format/builds/7646/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7747cb14a86dba4def878d1ef176ff42f8f274d">f7747cb14a86dba4def878d1ef176ff42f8f274d</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-50406-cms-fds-renderers">LPD-50406-cms-fds-renderers</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/3a21d44e64e758b7707d3a237d0c7adac67b3e00">3a21d44e64e758b7707d3a237d0c7adac67b3e00</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-source-format/7634/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/7634/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-source-format/7646/">test-portal-source-format#7646</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-source-format/builds/7646/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6444">liferay-content-management#6444</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/160424683">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6444 - 2025-03-10[01:20:08]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>160424683</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-39.liferay.com/job/publish-testray-report/27769/">publish-testray-report#27769</a><br/></div></div>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-source-format/7634/">test-portal-source-format#7634</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/7634/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6444">liferay-content-management#6444</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/160425405">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6444 - 2025-03-10[01:20:47]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>160425405</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/publish-testray-report/28038/">publish-testray-report#28038</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 24 out of 24 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 31 out of 31 jobs passed in 1 hour</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7747cb14a86dba4def878d1ef176ff42f8f274d">f7747cb14a86dba4def878d1ef176ff42f8f274d</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7747cb14a86dba4def878d1ef176ff42f8f274d">f7747cb14a86dba4def878d1ef176ff42f8f274d</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/159886880">EE Development Acceptance (master) - 1222 - 2025-03-08[12:36:19]</a></p><details><summary><strong>ci:test:stable - 24 out of 24 jobs PASSED</strong></summary><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li>central-requirements<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199954/">central-requirements/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/778557/">functional-smoke-tomcat90-mysql57_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57_stable<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1031468/">functional-upgrade-tomcat90-mysql57_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1025450/">functional-upgrade-tomcat90-mysql57_stable/0/1</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1024748/">functional-upgrade-tomcat90-mysql57_stable/0/2</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1082700/">js-unit/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343677/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1212049/">modules-compile/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199989/">modules-compile/0/1</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128540/">modules-compile/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091054/">modules-compile/0/3</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1082729/">modules-compile/0/4</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1031467/">modules-compile/0/5</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128539/">modules-compile/0/6</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1025451/">modules-compile/0/7</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1106181/">modules-compile/0/8</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1212048/">modules-compile/0/9</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091053/">modules-compile/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/778558/">modules-compile/0/11</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1161628/">modules-compile/0/12</a></li></ul></li><li>modules-integration-mysql57_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1298719/">modules-integration-mysql57_stable/0/0</a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1161627/">modules-unit_stable/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128537/">playwright-js-tomcat90-mysql57_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091051/">unit_stable/0/0</a></li></ul></li></ul></details></details><details><summary><strong>ci:test:relevant - 31 out of 31 jobs PASSED</strong></summary><details><summary><strong>31 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12823/">test-portal-acceptance-pullrequest(master)</a></li><li>central-requirements<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199954/">central-requirements/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/778557/">functional-smoke-tomcat90-mysql57_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57_stable<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1031468/">functional-upgrade-tomcat90-mysql57_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1025450/">functional-upgrade-tomcat90-mysql57_stable/0/1</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1024748/">functional-upgrade-tomcat90-mysql57_stable/0/2</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1082700/">js-unit/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343677/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1212049/">modules-compile/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199989/">modules-compile/0/1</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128540/">modules-compile/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091054/">modules-compile/0/3</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1082729/">modules-compile/0/4</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1031467/">modules-compile/0/5</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128539/">modules-compile/0/6</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1025451/">modules-compile/0/7</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1106181/">modules-compile/0/8</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1212048/">modules-compile/0/9</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091053/">modules-compile/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/778558/">modules-compile/0/11</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1161628/">modules-compile/0/12</a></li></ul></li><li>modules-integration-mysql57<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128538/">modules-integration-mysql57/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091052/">modules-integration-mysql57/0/1</a></li></ul></li><li>modules-integration-mysql57_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1298719/">modules-integration-mysql57_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199955/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199988/">modules-unit/0/0</a></li></ul></li><li>modules-unit_stable<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1161627/">modules-unit_stable/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1199987/">playwright-js-tomcat90-mysql57/0/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1031466/">playwright-js-tomcat90-mysql57/1/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1128537/">playwright-js-tomcat90-mysql57_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1091051/">unit_stable/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12823/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12823//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/159719
[Console](https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12823//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12823/">test-portal-acceptance-pullrequest(master)#12823</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12823/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6444">liferay-content-management#6444</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/160472319">[master] ci:test:relevant - oliv-yu &gt; liferay-content-management - PR#6444 - 2025-03-10[01:20:47]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>160472319</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/publish-testray-report/28043/">publish-testray-report#28043</a><br/></div></div>
